libs/sdk-ui-kit/src/utils/featureFlags.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2021 GoodData Corporation
```

Comment:
missing unit tests

Reply:
I added unit test for those function thanks